internal/services/oracle/oracle_network_anchor_data_source.go

```diff
@@ -0,0 +1,214 @@
+	DnsForwardingEndpointIpAddress     string `tfschema:"dns_forwarding_endpoint_ip_address"`
+	DnsForwardingRuleUrl               string `tfschema:"dns_forwarding_rule_url"`
+	DnsForwardingEndpointNsgRuleUrl    string `tfschema:"dns_forwarding_endpoint_nsg_rule_url"`
+	DnsListeningEndpointAllowedCidrs   string `tfschema:"dns_listening_endpoint_allowed_cidrs"`
+	DnsListeningEndpointIpAddress      string `tfschema:"dns_listening_endpoint_ip_address"`
+	DnsListeningEndpointNsgRuleUrl     string `tfschema:"dns_listening_endpoint_nsg_rule_url"`
+}
+
+func (d NetworkAnchorDataSource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ValidateFunc: validation.All(
+				validation.StringLenBetween(3, 24),
+				validation.StringMatch(regexp.MustCompile(`^[a-zA-Z0-9-]+$`), "Name may include letters, numbers, or hyphens only"),
+			),
+		},
+
+		"resource_group_name": commonschema.ResourceGroupNameForDataSource(),
+	}
+}
+
+func (d NetworkAnchorDataSource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.LocationComputed(),
+
+		"resource_anchor_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"provisioning_state": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"vnet_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"subnet_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"cidr_block": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"oci_vcn_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"oci_vcn_dns_label": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"oci_subnet_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"oci_backup_cidr_block": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"oracle_dns_forwarding_endpoint_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		},
+		"oracle_dns_listening_endpoint_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		},
+		"oracle_to_azure_dns_zone_sync_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Computed: true,
+		},
+		"dns_forwarding_rule_url": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"dns_forwarding_endpoint_ip_address": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"dns_listening_endpoint_allowed_cidrs": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"dns_listening_endpoint_nsg_rule_url": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"dns_listening_endpoint_ip_address": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"dns_forwarding_endpoint_nsg_rule_url": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"tags":  commonschema.TagsDataSource(),
+		"zones": commonschema.ZonesMultipleComputed(),
```

Comment:
Let's move the tags to the bottom of the attributes to follow the contributor documentation.
```suggestion
		"zones": commonschema.ZonesMultipleComputed(),
		"tags":  commonschema.TagsDataSource(),
```